protocol/0056-REWA-rewards_overview.md

```diff
@@ -130,6 +130,32 @@ If a party **is not** a consensus or standby validator, their reward metric is s
 
 $$m_v = 0$$
 
+### Realised Returns metric
+
+The realised returns metric, , $m_{rz}$, measures the returns a party has realised across a number of epochs.
+
+Let $rz_i$ be a parties realised returns in an epoch. At the start of each epoch, the network will set $rz_i$ to `0`.
+
+During the epoch, each parties realised returns will be incremented as follows:
+
+- a party pays or receives a funding payment
+
+$$rz_i = rz_i + fundingPayment$$
+
+- a party **reduces** their position
+
+$$rz_i = rz_i + (tradePrice - averageEntryPrice) * positionDelta$$
+
+At the end of the epoch, the average realised return metric over the last $N$ epochs is calculated as follows.
+
+Let:
+
+- $m_{rz}$ be the parties relative return reward metric
```

Comment:
shouldn't that be the party's realised Returns metric rather than relative returrn?

Reply:
Yep thanks, have spotted another one of these typos as well, will fix.